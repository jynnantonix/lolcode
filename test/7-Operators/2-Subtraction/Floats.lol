This test makes sure the subtraction operator works correctly on floating point decimal arguments.

HAI
	VISIBLE DIFF OF 1.23 AN 4.56 MKAY?
KTHXBAI
