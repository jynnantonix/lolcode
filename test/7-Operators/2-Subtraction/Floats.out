-3.3299999999999996 
